plugin/bufferline.vim

```diff
@@ -101,6 +101,7 @@ let icons = extend({
 \ 'bufferline_default_file': '',
 \ 'bufferline_separator_active':   '▎',
 \ 'bufferline_separator_inactive': '▎',
+\ 'bufferline_separator_tabpage': '▎',
```

Comment:
Why is it necessary to use a different icon here? IMHO the separator *should* be the same as the other separators.

Reply:
It isn't necessary, I misinterpreted one of the other comments and thought you were asking for it.

I'll get it removed.

Reply:
@romgrk I noticed when moving this out that `g:icons` isn't documented in the README or docs. Should I put something in there about it?

Reply:
> I put it in because of what was said in one of the other comments: what if someone just wants to adjust the BufferTapbageFill separator and not the others?

This is for hiding it right? Didn't mention but I thought this could be done by using the same fg & bg color for the `BufferTabpageFill` group.

> @romgrk I noticed when moving this out that the bufferline_* entries of g:icons aren't documented in the README or docs. Should I put something in there about it so people know how to customize it?

I don't know -_- I've been too lazy to expose that API properly, but I guess that yes it should be documented. Also if we're going to expose it as real options, it's probably better to keep it in `g:bufferline`, as `g:bufferline.icons`?
It was as `g:icons` initially because this was extracted from my config and I keep a big `g:icons` dictionary for all my icon needs.

Reply:
I suppose refactoring `g:icons` into `g:bufferline.icons` should be a separate PR… I don't mind putting a draft together for that if you don't mind checking it ;)